notebooks/nearest_neighbors.livemd

```diff
@@ -0,0 +1,255 @@
+<!-- livebook:{"file_entries":[{"name":"knn_gyms.png","type":"attachment"}]} -->
```

Comment:
Oh, you should also run the notebook with persist outputs enabled, so we have the outputs in the guides. :)